CMakeLists.txt

```diff
@@ -123,6 +123,44 @@ set(BOOST_DIRECTLY_REQUIRED_LIBRARIES filesystem system regex program_options)
 
 find_package(PkgConfig)
 
+if (MSVC)
+  # Disable distracting warning messages
+  add_definitions(/wd4003)  # not enough arguments for function-like macro invocation
+  add_definitions(/wd4005)  # macro redefinition
+  add_definitions(/wd4244)  # conversion, possible loss of data
+  add_definitions(/wd4267)  # '': conversion from '' to '', possible loss of data
+  add_definitions(/wd4305)  # '': truncation from '' to ''
+  add_definitions(/wd4996)  # '': This function or variable may be unsafe. Consider using [...]
+
+  # pkg-config is required to build OpenSCAD, but is not installed by default as a part of Windows.
+  # Installation instructions are available online e.g. as a comment posted by "E_net4 the curator":
+  #   https://stackoverflow.com/questions/1710922/how-to-install-pkg-config-in-windows
+  #
+  find_path(PKGCONFIG_EXE pkg-config.exe REQUIRED)
```

Comment:
This should NOT be REQUIRED, if you want the subsequent conditional to ever be reached.

Reply:
The reason why this is REQUIRED is in the build instructions.

Reply:
I think what that comment meant to say is that the following `if (NOT PKGCONFIG_EXE)` will not be reached with the `REQUIRED` in `find_path` as `cmake' will stop processing there already. So it will not give the custom message.

Reply:
Ouch, sorry; I am blind. :(

Fixed and pushed in 49191e57c7f76405dbadd19776196cc578d71fec. I can't test i,t because qt5 won't build for me at the moment, but hopefully it's a simple-enough change.

Reply:
No worries, we build test all the current platforms. As long as those go fine, adding this is an improvement even if there's some small glitches we may find later. It's great to get VS suport.